ydb/library/yql/sql/v1/format/sql_format.cpp

```diff
@@ -891,27 +891,24 @@ friend struct TStaticData;
         Visit(msg.GetToken1());
         Visit(msg.GetBlock2());
         Visit(msg.GetBlock3());
-        Visit(msg.GetRule_simple_table_ref4());
-        Visit(msg.GetToken5());
+        Visit(msg.GetBlock4());
```

Comment:
We need to add tests for this new behaviour to `ydb/library/yql/sql/v1/format/sql_format_ut.cpp`